SimTKcommon/Geometry/src/DecorativeGeometryRep.h

```diff
@@ -587,7 +595,7 @@ void implementGeometry(DecorativeGeometryImplementation& geometry) const overrid
 SimTK_DOWNCAST(DecorativeMeshFileRep, DecorativeGeometryRep);
 private:
 std::string meshFile;
-
+std::shared_ptr<PolygonalMesh> pmesh;
```

Comment:
It is true that PolygonalMesh works as is but it doesn't  or I couldn't find a method to tell if it has been populated/initialized. If you believe a separate flag (isInitialized) is cleaner  I'll add that and remove the pointer.